### PR TITLE
[dev] AB#32319 AB#32320 AB#32321 AB#32322 - Reorganized Components In `./docs`

### PR DESCRIPTION
**Pull request by morethanfire**

### CODE REVIEW

File: docs/src/dataDisplay/headerDeprecated/headerDeprecated.jsx [hunk 1]
```diff
@@ -6,10 +6,10 @@ import {
 } from 'react-cm-ui';
```
Comment (groberts314) [inline docs/src/dataDisplay/headerDeprecated/headerDeprecated.jsx:6]: The component is still `<Header>` not `<HeaderDeprecated>` ... are there near future plans to deprecate it?

File: docs/src/inputs/dropdownDeprecated/dropdownDeprecated.jsx [hunk 2]
```diff
@@ -17,10 +17,10 @@ import { Card, Dropdown, Grid, Header, SubNavigation, TitleBar } from 'react-cm-
 import { Link } from 'react-router';
```
Comment (groberts314) [inline docs/src/inputs/dropdownDeprecated/dropdownDeprecated.jsx:17]: Are we planning on renaming `<Dropdown>` => `<DropdownDeprecated>`?

File: docs/src/layout/containerDeprecated/containerDeprecated.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import {
```
Comment (groberts314) [inline docs/src/layout/containerDeprecated/containerDeprecated.jsx:1]: Are we planning on changing `<Container>` to `<ContainerDeprecated>`?
This deprecation I think I at least expected ( `<Header>` and `<Dropdown>` caught me a little bit off-guard).

File: docs/src/navigation/subNavigationDeprecated/subNavigationDeprecated.js [hunk 4]
```diff
@@ -1,15 +1,13 @@
-
 import MediaQuery from 'react-responsive';
```
Comment (groberts314) [inline docs/src/navigation/subNavigationDeprecated/subNavigationDeprecated.js:1]: Are we planning on renaming the component to indicate it is deprecated?

File: docs/src/routes.jsx [hunk 5]
```diff
@@ -11,495 +16,39 @@ const routes = (
...
+                )) : map(levelOne.items, (levelOneItem) => (
+                    <Route
+                        getComponent={(location, callback) => {
+                            import(`./${levelOne.directory}/${levelOneItem.component}`)
+                                .then((module) => callback(null, module.default));
+                        }}
+                        key={`${levelOneItem.path}`}
+                        path={levelOneItem.path}
+                    />
+                ))}
```
Comment (groberts314) [inline docs/src/routes.jsx:49]: Nice and clean!  I like!

File: docs/src/dataDisplay/headerDeprecated/headerDeprecated.jsx (hunk 1 above)
Comment (morethanfire) [reply docs/src/dataDisplay/headerDeprecated/headerDeprecated.jsx:6]: Yeah, it is a soft deprecation; bwahaha!
We should only be using the Typography component now.

File: docs/src/inputs/dropdownDeprecated/dropdownDeprecated.jsx (hunk 2 above)
Comment (morethanfire) [reply docs/src/inputs/dropdownDeprecated/dropdownDeprecated.jsx:17]: I don't think we should rename these in `./src`, but in `./docs/src` my thinking was/is to better identify what is deprecated. I could be wrong. Let me know.

File: docs/src/layout/containerDeprecated/containerDeprecated.jsx (hunk 3 above)
Comment (morethanfire) [reply docs/src/layout/containerDeprecated/containerDeprecated.jsx:1]: I don't think we should rename these in `./src`, but in `./docs/src` my thinking was/is to better identify what is deprecated. I could be wrong. Let me know.

File: docs/src/navigation/subNavigationDeprecated/subNavigationDeprecated.js (hunk 4 above)
Comment (morethanfire) [reply docs/src/navigation/subNavigationDeprecated/subNavigationDeprecated.js:1]: Hard deprecation. We should no longer be using this from here and out.

I don't think we should rename these in `./src`, but in `./docs/src` my thinking was/is to better identify what is deprecated. I could be wrong. Let me know.

File: docs/src/routes.jsx (hunk 5 above)
Comment (morethanfire) [reply docs/src/routes.jsx:49]: Yeah, I believe this will help with maintenance.

**Pull request merged**